dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -81,7 +81,7 @@ def summarize_progress
     if plc_course_unit.has_evaluation?
       Plc::LearningModule::MODULE_TYPES.select {|type| categories_for_stage.include?(type)}.each do |flex_category|
         module_category = flex_category
-        category_name = I18n.t("flex_category.#{module_category}")
+        category_name = I18n.t("flex_category.#{module_category}", default: I18n.t('flex_category.required'))
```

Comment:
I'm going with `flex_category.required` as the default here because it's the default type when creating a learning module:

https://github.com/code-dot-org/code-dot-org/blob/71ae4492da3ea5a8b8d519ef569b9606be29801b/dashboard/app/models/script.rb#L117

It's translated to "Overview" which is a nice generic title for a category:

https://github.com/code-dot-org/code-dot-org/blob/d393e1f9fe84524c5c78f1ffaae3488a1a39f8c7/dashboard/config/locales/en.yml#L886

And finally, the education team has only been using the "required" type for PLC courses for some time, as documented in https://github.com/code-dot-org/code-dot-org/pull/31015.